java/code/src/com/redhat/rhn/domain/server/VirtualInstanceFactory.java

```diff
@@ -96,8 +100,9 @@ public VirtualInstance lookupByGuestId(Org org, Long id) {
     public VirtualInstance lookupByGuestId(Long id) {
         Session session = HibernateFactory.getSession();
         return (VirtualInstance) session.getNamedQuery(
-                "VirtualInstance.lookupGuestBySid").
-                setParameter("sid", id).uniqueResult();
+                "VirtualInstance.lookupGuestBySid")
```

Comment:
Probably the line break before this line could be removed or the alignment fixed